src/System.Management.Automation/namespaces/FileSystemProvider.cs

```
@@ -5573,7 +5576,7 @@ protected override bool IsItemContainer(string path)
 
             path = NormalizePath(path);
 
-            return Utils.DirectoryExists(path);
+            return Directory.Exists(path);
```

same question, why we ignore all exceptions here?

---

I see. I guess it makes sense to swallow all exceptions here. #closed